### DISCUSSION TITLE
[Question]: Multiple RDS/Database instances

### DISCUSSION THREAD

**Discussion by thereisnotime** [movetokube/postgres-operator#193] [Q&A] [answered]
Hello,

I have been conducting experiments with various annotations and deployments, yet I am encountering difficulties in implementing multiple databases. 
Specifically, I am managing two external RDS PostgreSQL instances and aim to effectively administer both. 
My approach involved creating the operator twice in distinct namespaces, but this resulted in an error attributed to CRD duplication. Could someone kindly provide an example or guidance on utilizing the operator with multiple external databases?

**Edit:** I managed to get it to work by disabling the CRD via Helm of the second deployment and changing it's chart name via the values, also both are deployed to separate namespaces.

Wonder if there is a proper way to do that.

**Comment by hitman99** [comment, marked as answer]
I think you could use annotations to achieve what you intend. The information on readme.md is slim, but check it out: https://github.com/movetokube/postgres-operator?tab=readme-ov-file#multiple-operator-support

By setting `POSTGRES_INSTANCE` you can use multiple operators and segregate CRs based on that annotation.
